### ISSUE TITLE
Can't be installed with django 5.0

### ISSUE THREAD

**Issue by fabien-michel** [Pierre-Sassoulas/django-zxcvbn-password-validator#74]
The package is not marked to be compatible with django 5.0 so it can't be installed (at least with poetry)

```
Because no versions of django-zxcvbn-password-validator match >1.4.2,<2.0.0
 and django-zxcvbn-password-validator (1.4.2) depends on Django (>=2.0,<5), django-zxcvbn-password-validator (>=1.4.2,<2.0.0) requires Django (>=2.0,<5).
So, because e-audit depends on both django (~5.0) and django-zxcvbn-password-validator (^1.4.2), version solving failed.
```

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue, should be fixed in 1.4.3

**Comment by fabien-michel** [comment]
Wow, thank you for your quick response!
